features/server/file.include/etc/kernel/cmdline.d/90-lsm.cfg

```diff
@@ -0,0 +1 @@
+CMDLINE_LINUX="$CMDLINE_LINUX selinux=1 apparmor=0"
```

Comment:
Hmm, this does not qualify as "remove", as someone can expect from the commit message.

Reply:
True, the wording is unfortunate.